src/pages/CartPage.js

```diff
@@ -74,7 +74,7 @@ const CartPage = () => {
   const cartClasses = {
```

Comment:
This constant can be outside the component so it's not recreated on every render

Reply:
Ohh yes, thanks!